test/tests/nodes_tests.py

```diff
@@ -57,20 +57,40 @@ def __init__(self):
                 'name': 'test_compute_node',
                 'type': 'compute',
                 'obmSettings': [{
-                    'config': { 'host': '00:01:02:03:04:05', 'password': 'pass', 'user': 'user' },
+                    'config': {'host': '00:01:02:03:04:05', 'password': 'pass', 'user': 'user'},
                     'service': 'ipmi-obm-service'
                 }]
             }
         ]
 
-    @test(groups=['nodes.test','test-nodes'])
+    @test(groups=['nodes.test', 'test-nodes'])
     def test_nodes(self):
         """ Testing GET:/nodes """
         Nodes().api1_1_nodes_get()
         nodes = dumps(self.__client.last_response.data)
-        assert_equal(200,self.__client.last_response.status)
+        assert_equal(200, self.__client.last_response.status)
         assert_not_equal(0, len(nodes), message='Node list was empty!')
 
+    @test(groups=['create-node-id-obm-identify'], depends_on_groups=['nodes.test', 'test-nodes'])
+    def test_node_id_obm_identify_create(self):
+        """ Testing POST:/nodes/:id/obm/identify """
+        Nodes().api1_1_nodes_get()
+        nodes = loads(self.__client.last_response.data)
+        codes = []
+        data = {
+                    "value": True
+        }
+        for n in nodes:
+            if n.get('type') == 'compute':
+                uuid = n.get('id')
+                Nodes().api1_1_nodes_identifier_obm_identify_post(uuid, data)
+                rsp = self.__client.last_response
+                codes.append(rsp)
+        assert_not_equal(0, len(codes), message='Failed to find compute node Ids')
+        for c in codes:
+            assert_equal(200, c.status, message=c.reason)
+        assert_raises(rest.ApiException, Nodes().api1_1_nodes_identifier_get, 'fooey')
```

Comment:
Not critical, but I might suggest this last test as a separate test case because it's testing a different scenario than the previous bits of code creating nodes. If this broke, but not the rest of this method - it would be a tad more difficult to dig out, because we're testing the corner case.

(This is totally my opinion on structuring tests - take it as a suggestion only, not a requirement to change for merging)